spring-cloud-gcp-data-firestore/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Is this to enable mocking the `FirstoreStub`?

Reply:
Exactly! It is necessary to enable final class mocking.